src/chapters/orm-relationships/exercises.rst

```diff
@@ -0,0 +1,27 @@
+Exercises: The Early Bird Gets the ORM!
+=======================================
+
+Now that we have gotten very familiar with our ``coding-events`` application, let's design some additional features.
+As you work on your ``coding-events`` application, you may have been inspired by `Meetup <https://www.meetup.com/>`_!
+One of the cool features that Meetup has is that people can sign up for accounts.
+They can use their Meetup accounts to follow the events they are most interested in and keep track of their calendar of events.
+If you want to add similar features to ``coding-events``, you would need to add a ``Person`` class.
+For the exercises, answer the following questions about what your ``Person`` class would look like.
+
+.. admonition:: Note
+
+   You do not have to code anything to complete these exercises!
+   This is mainly focused on using our design skills to add a new feature to your application.
+
+#. You need to add a ``Person`` class to hold necessary info about users of our app. What fields and methods would this class hold?
+#. Would you need to add any additional classes to ``Person`` to make the app work?
```

Comment:
Since this is a yes/no question, it should probably be followed by a secondary prompt. Something like, "If so, what classes would be necessary?"

Reply:
Done!